lib/id3c/cli/redcap.py

```diff
@@ -369,6 +369,50 @@ def metadata(self) -> List[Dict[str, str]]:
         return self._fetch('metadata', {})
 
 
+    def update_metadata(self, metadata: Dict[str, str]) -> int:
```

Comment:
Given the existing `.fields` property, I think this should be renamed to `.update_fields()`.